tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -241,6 +243,36 @@ var _ = Describe("Create machinepool",
 				}
 			})
 
+		It("List instance-types with region flag - [id:72174]",
+			labels.Low, labels.Runtime.OCMResources,
+			func() {
+				By("List the available instance-types with the region flag")
+				typesList := []string{"dl1.24xlarge", "g4ad.16xlarge", "c5.xlarge"}
+				region := "us-east-1"
+				accountRolePrefix := fmt.Sprintf("QEAuto-accr72174-%s", time.Now().UTC().Format("20060102"))
+				_, err := ocmResourceService.CreateAccountRole("--mode", "auto",
+					"--prefix", accountRolePrefix,
+					"--permissions-boundary", permissionsBoundaryArn,
+					"-y")
+				Expect(err).To(BeNil())
+				accountRoleList, _, err := ocmResourceService.ListAccountRole()
+				Expect(err).To(BeNil())
+				classicInstallerRoleArn := accountRoleList.InstallerRole(accountRolePrefix, false).RoleArn
+				availableMachineTypes, _, err := ocmResourceService.ListInstanceTypes(
+					"--region", region, "--role-arn", classicInstallerRoleArn)
+				Expect(err).To(BeNil())
+				var availableMachineTypesIDs []string
+				for _, it := range availableMachineTypes.InstanceTypesList {
+					availableMachineTypesIDs = append(availableMachineTypesIDs, it.ID)
+				}
+				Expect(availableMachineTypesIDs).To(ContainElements(typesList))
+
+				availableMachineTypes, output, err := ocmResourceService.ListInstanceTypes(
+					"--region", "xxx", "--role-arn", classicInstallerRoleArn)
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("ERR: Unsupported region 'xxx', available regions"))
+			})
+
```

Comment:
Is it a bug for the error message? What does `ERR: Unsupported region 'xxx'`, and then `available regions` mean?

Reply:
Since it is an ocm-resources type rather than Day1/2, I figured that HCP testing was not necessary. However, if you think this is needed, I can update the test case and add that. 

@yasun1 This was my understanding as well since it is only an unsupported region check.

Reply:
It is a little redundant to do the testing with `--hosted-cp`. But from the product and customer usage sides, `--hosted-cp` could be the most popular usage, so I'd like you to add a new checkpoint. Thanks.

> This was my understanding as well since it is only an unsupported region check.

From my understanding, `Unsupported region` could be enough.

Reply:
I'll add a new test case and open it as a separate PR. This test case is encased by `By("Skip testing if the cluster is not a Classic cluster")
			isHostedCP, err := rosaClient.Cluster.IsHostedCPCluster(clusterID)
			Expect(err).ToNot(HaveOccurred())
			if isHostedCP {
				SkipNotClassic()
			}`

Reply:
I think this case is an OCMResource case and does not need to prepare a cluster. We just need to add a checkpoint to validate the option '--hosted-cp'.